### PR TITLE
Use admin user to install Kogito Operator for tests

### PR DESCRIPTION
**Pull request by sutaakar**
The ordinary user doesn't have rights for this.
@ricardozanini Could you please review?

### CODE REVIEW

File: openshift-example-tests/src/main/java/org/kogito/examples/openshift/deployment/OperatorDeployer.java [hunk 1]
```diff
@@ -53,29 +54,40 @@
...
+            project.getAdmin().customResourceDefinitions().create(customResourceDefinition);
+        }
+    }
+
     private static void createServiceAccountInProject(Project project) {
         logger.info("Creating service account in project '" + project.getName() + "' from " + OPERATOR_SERVICE_ACCOUNT);
-        ServiceAccount serviceAccount = project.getMaster().serviceAccounts().load(getResource(OPERATOR_SERVICE_ACCOUNT)).get();
-        project.getMaster().createServiceAccount(serviceAccount);
+        ServiceAccount serviceAccount = project.getAdmin().serviceAccounts().load(getResource(OPERATOR_SERVICE_ACCOUNT)).get();
+        project.getAdmin().createServiceAccount(serviceAccount);
```
Comment (ricardozanini) [inline openshift-example-tests/src/main/java/org/kogito/examples/openshift/deployment/OperatorDeployer.java:78]: The Operator Service Account will be created within the Operator https://issues.jboss.org/browse/KOGITO-66. I understand that you need this now, but you should remember to remove it as soon as I fix this JIRA. :p

File: openshift-example-tests/src/main/java/org/kogito/examples/openshift/deployment/OperatorDeployer.java [hunk 2]
```diff
@@ -53,29 +54,40 @@
...
     }
 
-    private static void createRoleInProject(Project project, OpenShiftBinary masterBinary) {
+    private static void createRoleInProject(Project project) {
         logger.info("Creating role in project '" + project.getName() + "' from " + OPERATOR_ROLE);
-        masterBinary.execute("create", "-f", OPERATOR_ROLE);
+        Role role = project.getAdmin().rbac().roles().load(getResource(OPERATOR_ROLE)).get();
+        project.getAdmin().rbac().roles().inNamespace(project.getName()).create(role);
```
Comment (ricardozanini) [inline openshift-example-tests/src/main/java/org/kogito/examples/openshift/deployment/OperatorDeployer.java:84]: See above, same for the bindings. We could end up with duplicated content. Or this user need to be created to run the Operator?

**Review by ricardozanini — CHANGES_REQUESTED** [review]

File: openshift-example-tests/src/main/java/org/kogito/examples/openshift/deployment/OperatorDeployer.java (hunk 2 above)
Comment (sutaakar) [reply openshift-example-tests/src/main/java/org/kogito/examples/openshift/deployment/OperatorDeployer.java:84]: These steps are actually required to install Kogito Operator itself. They are not related to actions done by Kogito Operator when building an application using CRs.
This class is responsible for installation of Kogito Operator. Current approach is to install it "manually" by creating all needed object by ourself - as Kogito operator is not yet published in catalog.

Comment (ricardozanini) [reply openshift-example-tests/src/main/java/org/kogito/examples/openshift/deployment/OperatorDeployer.java:84]: Sure!

**Review by ricardozanini — APPROVED** [review]

**Review by mswiderski — APPROVED** [review]

**Pull request merged**